ntoskrnl/include/internal/spinlock.h

```diff
@@ -6,50 +6,25 @@
 * PROGRAMMERS:     Alex Ionescu (alex.ionescu@reactos.org)
 */
 
+#if DBG && defined(_M_IX86) && defined(CONFIG_SMP)
 VOID
 NTAPI
 Kii386SpinOnSpinLock(PKSPIN_LOCK SpinLock, ULONG Flags);
```

Comment:
Why is this going under `DBG` or `CONFIG_SMP`? This function seems to be exported on the x86 free build.